hexa/connections/graphql/schema.graphql

```diff
@@ -0,0 +1,29 @@
+"""
+DHIS2 metadata item
+"""
+type DHIS2MetadataItem {
+    id: String!     # Unique identifier
+    name: String!   # Name of the metadata item
+}
+enum DHIS2ConnectionError {
+    CONNECTION_ERROR
+    UNKNOWN_ERROR
+}
+"""
+DHIS2 metadata query result
+"""
+type DHIS2QueryResult {
+    data: [DHIS2MetadataItem!]  # List of metadata items
+    errors: [String!]!      # List of errors
+}
```

Comment:
I have seen that usually, we use a `success` boolean as well (https://github.com/BLSQ/openhexa-app/blob/d9d7159d6fc6d1982ca6f4067fb04c878dda3ceb/hexa/files/graphql/schema.graphql#L117 and https://github.com/BLSQ/openhexa-app/blob/d9d7159d6fc6d1982ca6f4067fb04c878dda3ceb/hexa/files/graphql/schema.graphql#L142)